docs/resources/mls_instance_v1.md

```diff
@@ -75,91 +75,105 @@ resource "flexibleengine_mls_instance_v1" "instance" {
   Changing this creates a new instance.
 
-* `public_ip` - (Required) Specifies the IP address of the instance. The structure is
-  described below. Changing this creates a new instance.
+* `public_ip` - (Required, List, ForceNew) Specifies the IP address of the instance. The [public_ip](#mls_public_ip)
+  object structure is documented below. Changing this creates a new instance.
 
+<a name="mls_public_ip"></a>
 The `public_ip` block supports:
 
-* `bind_type` - (Required) Specifies the bind type. Possible values: `auto_assign` and
+* `bind_type` - (Required, String, ForceNew) Specifies the bind type. Possible values: `auto_assign` and
   `not_use`. Changing this creates a new instance.
 
+<a name="mls_mrs_cluster"></a>
 The `mrs_cluster` block supports:
 
-* `id` - (Required) Specifies the ID of the MRS cluster. Changing this creates a new instance.
+* `id` - (Required, String, ForceNew) Specifies the ID of the MRS cluster. Changing this creates a new instance.
 
-* `user_name` - (Optional) Specifies the MRS cluster username. This parameter is mandatory
+* `user_name` - (Optional, String) Specifies the MRS cluster username. This parameter is mandatory
   only when the MRS cluster is in the security mode. Changing this creates a new instance.
 
-* `user_password` - (Optional) Specifies the password of the MRS cluster user. The password
+* `user_password` - (Optional, String) Specifies the password of the MRS cluster user. The password
   and username work in a pair. Changing this creates a new instance.
 
-## Attributes Reference
-
-The following attributes are exported:
-
-* `region` - See Argument Reference above.
-* `name` - See Argument Reference above.
-* `version` - See Argument Reference above.
-* `agency` - See Argument Reference above.
-* `flavor` - See Argument Reference above.
-* `network/vpc_id` - See Argument Reference above.
-* `network/subnet_id` - See Argument Reference above.
-* `network/security_group` - See Argument Reference above.
-* `network/available_zone` - See Argument Reference above.
-* `network/public_ip/bind_type` - See Argument Reference above.
-* `network/public_ip/eip_id` - Indicates the EIP ID, This is returned only when bind_type is
-  set to auto_assign.
-* `mrs_cluster` - See Argument Reference above.
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
 * `status` - Indicates the MLS instance status.
+
 * `inner_endpoint` - Indicates the URL for accessing the instance. Only machines in the same
   VPC and subnet as the instance can access the URL.
+
 * `public_endpoint` - Indicates the URL for accessing the instance. The URL can be accessed
   from the Internet. The URL is created only after the instance is bound to an EIP.
+
 * `created` - Indicates the creation time in the following format: yyyy-mm-dd Thh:mm:ssZ.
+
 * `updated` - Indicates the update time in the following format: yyyy-mm-dd Thh:mm:ssZ.
```

Comment:
Pls move `create` and `update` after `network`.

Reply:
The `network` field has two level nesting. It is batter to put them end of the document for the convenience of users' reference.

Reply:
It doesn't make sense, usually `create` and `update ` at the end.

BTW, `public_ip` and `eip_id` can be written as Level 2 or Level 3, as follows:

* `network` - Specifies the instance network information. 
  + `public_ip` - Specifies the IP address of the instance. 
    + `eip_id` - Indicates the EIP ID, This is returned only when bind_type is set to auto_assign.

Reply:
I know that the time fields such as create is placed last usually,but not certainly. It can be handled in a special way when meet special scene.
As for nested list style, the document is already in block style. One style is easier to accept by reader.

Reply:
1. Verify that xx should use the same valid value from creation to query, not 0.
2. The default value is 0. What is the meaning of what you display? A value of 0 is invalid.
3. This writing format is incorrect and does not comply with the standards.